### PR TITLE
fix wrong version in version-date when building git with merges of old commits on top

### PR DESCRIPTION
**Pull request by ailin-nemui**

### CODE REVIEW

**Comment by ailin-nemui** [comment]
@irssi/developers

File: utils/irssi-version.sh [hunk 1]
```diff
@@ -1,6 +1,6 @@
 #!/bin/sh
 
-DATE=`GIT_DIR=$1/.git git log -1 --pretty=format:%ai HEAD 2>/dev/null`
+DATE=`GIT_DIR=$1/.git git log -1 --pretty=format:%ci HEAD 2>/dev/null`
```
Comment (berkerpeksag) [inline utils/irssi-version.sh]: This LGTM but we can use the following commands to get date and time without doing any parsing on our end:

```sh
$ git log -1 --pretty=format:%cd --date=format:%Y-%m-%d
2019-07-17
$ git log -1 --pretty=format:%cd --date=format:%H:%M:%S
23:28:15
```

Comment (ailin-nemui) [reply utils/irssi-version.sh]: Nice. however the time mustn't start with 0 so I'm not sure if we can safely skip the awk

Comment (ailin-nemui) [reply utils/irssi-version.sh]: @dwfreed commented to just turn all those constants into strings

**Comment by ailin-nemui** [comment]
I'm merging this now let's keep the other improvements in mind

**Pull request merged**